### DISCUSSION TITLE
Tutorial No. 13 failed to compile

### DISCUSSION THREAD

**Discussion by KingsonKai** [eunomia-bpf/bpf-developer-tutorial#119] [Q&A]
<img width="988" alt="image" src="https://github.com/eunomia-bpf/bpf-developer-tutorial/assets/27922190/a4571ff9-2941-40aa-9ef2-f5b07516f424">
How to solve this error? I am using the main branch

**Comment by yunwei37** [comment]
Maybe you need a newer compiler?

What's your compiler version?
